ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/navigation-right/navigation-right.tsx

```diff
@@ -33,15 +33,14 @@ export interface Props {
   isGuest: boolean
 }
 
-const navigationRightUserIcon = '1.375rem'
+const navigationRightUserIcon = '1rem'
 const unseenNotifications = keyframes`
     0% {
         opacity: ${props => props.theme.minimumOpacity};
         transform: scale(1);
```

Comment:
If we're removing the enlargement, we can cut out the transform in this part as well.  It can be limited to just a flashing opacity.